cve_bin_tool/output_engine/__init__.py

```diff
@@ -76,13 +76,10 @@ def output_pdf(all_cve_data: Dict[ProductInfo, CVEData], outfile):
         + str(cvedb_data.get_cve_count())
         + " entries."
     )
-    pdfdoc.heading(1, "List of Identified Vulnerabilities")
-    pdfdoc.paragraph(
-        "The following vulnerabilities are reported against the identified versions of the libraries."
-    )
+    pdfdoc.heading(1, "List of All Scanned binaries")
     pdfdoc.createtable(
         "Vulnlist",
```

Comment:
With this change it's no longer a "Vulnlist", it is "Productlist" as you list all detected product versions.